jdaviz/core/helpers.py

```diff
@@ -407,40 +407,16 @@ def show_in_new_tab(self, title=None):  # pragma: no cover
                       DeprecationWarning)
         return self.show(loc="sidecar:tab-after", title=title)
 
-    def get_data(self, data_label=None, cls=None, subset_to_apply=None, statistic=None):
-        """
-        Returns data with name equal to data_label of type cls with subsets applied from
-        subset_to_apply.
-
-        Parameters
-        ----------
-        data_label : str, optional
-            Provide a label to retrieve a specific data set from data_collection.
-        cls : `~specutils.Spectrum1D`, `~astropy.nddata.CCDData`, optional
-            The type that data will be returned as.
-        subset_to_apply : str, optional
-            Subset that is to be applied to data before it is returned.
-        statistic : {'minimum', 'maximum', 'mean', 'median', 'sum'}, optional
-            The statistic to use to collapse the dataset.
-
-        Returns
-        -------
-        data : cls
-            Data is returned as type cls with subsets applied.
-
-        """
-        if self.app.config != "cubeviz" and statistic:
-            raise AttributeError(f"{self.app.config} does not need the statistic parameter set.")
-
-        list_of_valid_statistic_values = ['minimum', 'maximum', 'mean',
-                                          'median', 'sum']
-        if statistic and statistic not in list_of_valid_statistic_values:
-            raise ValueError(f"statistic {statistic} not in list of valid"
-                             f" statistic values {list_of_valid_statistic_values}")
+    def _get_data(self, data_label=None, cls=None, subset_to_apply=None, function=None):
+        list_of_valid_function_values = ['minimum', 'maximum', 'mean',
+                                         'median', 'sum']
```

Comment:
Should we add "None" here?

Reply:
Same question. Also should it be a tuple?